### DISCUSSION TITLE
Dudas HTML

### DISCUSSION THREAD

**Discussion by OscarIslasH** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#241] [Q&A] [answered]
Apenas inicie con la practica de HTML y no se si todo lo que tenia que hacer ya lo hice, ¿No importa que no haya usado CSS? 
Vi que algunos explorers usaron CSS para mejorar la pagina pero no se usarlo. 

No se si asi este bien tomando en cuenta todo lo que la descripcion del repositorio pide
![Captura de pantalla 2022-03-04 011150](https://user-images.githubusercontent.com/99118114/156717151-0a2759fa-fcec-4938-9952-a14434d59a85.png)

**Comment by doguedogue** [comment, marked as answer]
Se ve bien 🚀
Aunque todo puede mejorarse no? Hay mucha información en youtube o tutoriales de html y css, vamos tu puedes
https://www.w3schools.com/

**Reply by OscarIslasH** [reply]
Soy nuevo en todo lo relacionado a HTML y CSS ademas que anduve con el tiempo encima, gracias por el apoyo, voy a tratar de mejorarlo lo mejor posible, animos igualmente y muchisimas gracias por la pagina ! Estoy seguro de que me va a servir bastante en todo el viaje.

**Comment by JuanSalvador73** [comment]
Los requerimientos solo piden la maquetación del sitio en HTML. Sin embargo muchos exploradores además de los recientes videos publicados en CSS ya cuentan con un poco de conocimiento en estilos, por eso los han integrado.

Entre mas completo entregues tu sitio, mejor, pero como lo tienes ahora cumple perfectamente los requerimientos. 
Animo y suerte :D

**Reply by OscarIslasH** [reply]
Gracias por aclararme lo pregunto mas que nada por que como hay que entregarlo mañana, tenia esa duda, ando con el tiempo encima.
Voy a ver los directos, videos y si tienes algun consejo o pagina que conozcas para facilitar alguna cosa como los juegos que pasaron ayer, te agradeceria si gustas compartidlos :D

**Comment by hackit92** [comment]
Como ya los compañeros te comentan arriba es una practica de HTML así que ya cumpliste (solo no olvides la sección de inventario y pedidos para el pastelero). Por otro lado te recomiendo en tiempos libres buscar más sobre HTML5 y CSS tanto en youtube como en otros lados (te recomiendo faztweb.com) más que nada para que vayas aprendiendo los alcances de cada uno y su integración futura con angular, react, js, etc. Ánimo explorer! Apenas vamos empezando :D
